AUTHORS.md

```diff
@@ -13,4 +13,3 @@
 ## Contributors
 
 * Trevor James Smith <smith.trevorj@ouranos.ca> [@Zeitsperre](https://github.com/Zeitsperre)
-* André Della Libera Zanchetta <andre.dellaliberazanchetta@umanitoba.ca> [@adlzanchetta](https://github.com/adlzanchetta)
```

Comment:
Hi @adlzanchetta, Just to confirm, you intended to remove yourself from here?

Reply:
Hi @Zeitsperre . The project has contributors listed in the ```.zenodo.json``` and in the ```AUTHORS.md``` files. As the first file has many more names than the second on, I understood that the the first file holds everyone who contributed to the project, regardless the size of the contribution, while the second lists the main/major authors. Thus, I indeed removed myself as it seemed more appropriate to have my name just in the ```.zenodo.json``` file as so far my contributions were very limited. If this is a misinterpretation and you find it is fine to keep my name in the ```AUTHORS.md```, I am fine with it too.

Reply:
I think (ideally) these two lists should be synchronized. The reasoning for the Zenodo list being longer had to do with a need to provide a Zenodo reference for a journal article a few months ago. The immediate need didn't include the `AUTHORS.md` and didn't have much time to also gather the email and GitHub usernames for those people, so the `AUTHORS.md` was left alone. I should create an issue to perform this synchronization, though.

Please feel free to leave your name (and your contact info, optionally).

Reply:
Thanks for the clarification, @Zeitsperre . I've commited a change to re-include my name and contact info there.